cpg_utils/hail_batch.py

```diff
@@ -6,8 +6,10 @@
 import os
 import tempfile
 import textwrap
+import uuid
```

Comment:
Thought that crazy linter would want this after `from typing …` but looks like it's me who needs a refresher on which way around `import AB` and `from AA import X` are supposed to go…

Reply:
Yeah, afaik it's python builtin `import X`, `from X`, then the same for external libs, then for local/private libs. I'm still not sure if capitalised should come before lower case, I just tweak it until the linter gives me the green light 😆